wcpredictor/data/confederations.csv

```diff
@@ -210,3 +210,108 @@ Hungary,UEFA
 Azerbaijan,UEFA
 Netherlands,UEFA
 San Marino,UEFA
+Andalusia,Unaffiliated
```

Comment:
Do we need these unaffiliated teams?    Are we not just excluding them anyway in `get_results_data`?

Reply:
I added them to give the option of running on the full dataset (though it likely doesn't make much difference).